packages/tomls/src/omnibus/reya_cronos.toml

```diff
@@ -17,7 +17,7 @@ include = [
     "../sbt/testnet.toml",
 ]
 
-version = "1.0.101"
+version = "1.0.103"
```

Comment:
used 2 versions to test if we can deploy tokens from omnibus, but gave up on the idea
keeping the version up-todate